VERSION

```diff
@@ -1 +1 @@
-3.3.0dev
+3.3.0
```

Comment:
I see 3.4.0 above, and 3.3.0 here.  Should they be the same?

Reply:
That mean it wasn't correctly updated on master when rolling to track the next release. Need to make sure to update it correctly next time. Pushing a fix for this branch now.